src/lib/core/data-table/data-table.ts

```diff
@@ -161,7 +145,7 @@ export class CdkTable<T> implements CollectionViewer {
   }
 
   ngOnDestroy() {
-    this._onDestroy.next();
+    this._onDestroy.complete();
```

Comment:
Looks like the rxjs docs will eventually be updated to reflect behavior being inconsistent with reactivex.io https://github.com/ReactiveX/rxjs/issues/2160

Also don't think complete() is required, but as far as I've seen, [this SO answer](https://stackoverflow.com/a/41177163/4811678) has been the recommended approach.